### ISSUE TITLE
raiser.c should be updated with Cython 0.24

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8463]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8463|
|Type|enhancement|
|Created|2016-06-13 09:41:14Z|

raiser.c (which is generated from raiser.pyx) should be updated with Cython 0.24,
which removes support for Python < 2.6,
and has better support for Python >= 3.2.

<details><summary>Searchable metadata</summary>

```
trac-id__8463 8463
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1465810874008976 1465810874008976
changetime__1466069112084524 1466069112084524
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/183

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 4b2dbdd0e4f2b0bd4882c902be8aba9ad11f3280
```
#!CommitTicketReference repository="" revision="4b2dbdd0e4f2b0bd4882c902be8aba9ad11f3280"
Merge 8463-rodrigc-raiser: Update raiser.c using cython 0.24

Author: rodrigc
Reviewer: hawkowl
Fixes: #8463
```
